### PR TITLE
Make store link use locale-specific slug for Swedish

### PR DESCRIPTION
**Pull request by alebedev**
## Describe your changes

Quick and dirty solution for showing /se/forsakringar but /en-se/store

## Checklist before requesting a review

- [ ] I have performed a self-review of my code

### CODE REVIEW

**Comment by alebedev** [comment]
Current dependencies on/for this PR:
* main
  * **PR #1751** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/1751" target="_blank">Graphite</a> 
    * **PR #1762** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/1762" target="_blank">Graphite</a> 
      * **PR #1764** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/1764" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/HedvigInsurance/racoon/1764?utm_source=stack-comment).

File: apps/store/src/utils/PageLink.ts [hunk 1]
```diff
@@ -21,8 +20,13 @@ const localePrefix = (locale?: RoutingLocale) => (locale ? `/${locale}` : '')
 
 export const PageLink = {
   home: ({ locale }: BaseParams = {}) => localePrefix(locale) || '/',
-  store: ({ locale }: BaseParams = {}) => `${localePrefix(locale)}/store`,
-  product: ({ locale, slug }: ProductPage) => `${localePrefix(locale)}/products/${slug}`,
+  store: ({ locale }: Required<BaseParams>) => {
```
Comment (robinandeer) [inline apps/store/src/utils/PageLink.ts:23]: nit: add a TODO comment here so we don't forget about fixing it

**Review by robinandeer — APPROVED** [review]
I think you came up with a nice solution given the circumstances :)

**Comment by alebedev** [comment]
⏳ This pull request is set to merge as part of a Graphite merge job
Job ID: ZJkndhg2wa4JObWNCmFs.
See details on [graphite.dev](https://app.graphite.dev/github/pr/HedvigInsurance/racoon/1764)

**Pull request merged**